<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hbSearchDialog</class>
 <widget class="QDialog" name="hbSearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>H-Bonds Search</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>491</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>491</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Streams</string>
   </property>
   <widget class="QComboBox" name="sCombo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Stream Source:</string>
    </property>
    <property name="buddy">
     <cstring>sCombo</cstring>
    </property>
   </widget>
   <widget class="QGroupBox" name="trajectoryRange">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Trajectory Range</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;End Index:</string>
     </property>
     <property name="buddy">
      <cstring>eIndex</cstring>
     </property>
    </widget>
    <widget class="QSpinBox" name="eIndex">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
     <property name="value">
      <number>9999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sIndex">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Start Index:</string>
     </property>
     <property name="buddy">
      <cstring>sIndex</cstring>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>491</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Significance Criterion</string>
   </property>
   <widget class="QCheckBox" name="cbEnergyTreshold">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>451</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ignore Bonds with Absolute Energy Value Less Than:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinEnergy">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>kcal/mol</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbPercentTreshold">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>451</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ignore Bonds Existing in Less Than:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinPercent">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string>%</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>90</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>of the Trajectory</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>491</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Search Options</string>
   </property>
   <widget class="QCheckBox" name="cbGrouping">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>26</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Group Bonds Formed by Equivalent Donor and/or Acceptor</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>sCombo</tabstop>
  <tabstop>trajectoryRange</tabstop>
  <tabstop>sIndex</tabstop>
  <tabstop>eIndex</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>hbSearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbEnergyTreshold</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinEnergy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbEnergyTreshold</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbPercentTreshold</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinPercent</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbPercentTreshold</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
